src/clarity-angular/dropdown/dropdown-menu.ts

```diff
@@ -4,6 +4,11 @@ import {Component} from "@angular/core";
     selector: "clr-dropdown-menu",
     template: `
         <ng-content></ng-content>
-    `
+    `,
+    host: {
+        "[class.dropdown-menu]" : "true",
```

Comment:
This class should probably be `clr-dropdown-menu` to bring it into line with our current path with CSS naming conventions.